### DISCUSSION TITLE
Severe end choked flutes

### DISCUSSION THREAD

**Discussion by stanleyloftness** [edwardkort/WWIDesigner#92] [NAF Study] [answered]
Edward,

I decided to try make some flutes with severe end shoes like a quena, shakuhachi,or anasazi flute.  I had no trouble designing them on the program, and made a G F# and F flute.  The F and F# flute for example goes from .875 to .5 inch for the last 1/2 inch.  After I built them, when cut to length they all ended up a half step below what was predicted.  Any idea what is going on?

Stan

**Comment by edwardkort** [comment]
Hey Stan,

Let’s try something simple first. Take one of the flutes you just made, with or without holes, and create/modify an instrument file with the as-built dimensions. Then measure the frequency that flute plays with all holes closed. Modify a tuning file (with the correct number of holes) and the measured frequency. 

Now run the fipple factor optimizer. I suspect that you are blowing softer with this choked flute than you do with your other flutes. 

Let me know your results, and we’ll go from there. 

Cheers,
Edward

**Comment by stanleyloftness** [comment]
I ran a fipple factor one my F# flute with no holes, which is actually a F flute.  It ended up being .599.  There is something about that severe an end taper that altering the acoustical prosperities.  All three flutes ended up actually being 100 cents flat.
<img width="1251" alt="F#4  875- 5 severe choke" src="https://user-images.githubusercontent.com/12632948/88710678-3475c900-d0d4-11ea-827d-34691629c501.png">

**Comment by edwardkort** [comment]
If I understand your post, your normal fipple factor (0.750) went to 0.599 with the choked flute. Since the fipple factor is a catch-all parameter which includes geometry as well as blowing characteristics, this is not surprising. You should use this new "historical" fipple factor when designing other such choked flutes. But, since cutting the flute to yield an F# flute will cut off all of your choke, I suggest just making an F flute with it.

The real question is whether the tuning offset you are seeing is due to a reduced blowing pressure for the fundamental note. If so, that reduction is likely to disappear as you go up the scale (because the choke will no longer be in play). The NAF model may not accommodate these dramatic and discontinuous changes in blowing pressure, so start with well-undersized holes. After you have made your first flute, you might blueprint it (max and min playing frequencies for each note) according to the protocol for the Whistle model - which will better handle these pressure changes.

**Comment by stanleyloftness** [comment, marked as answer]
It turns out that when cutting down the flute to tune the fundamental, 1/4" changes the tuning one half step.  I built a F and F# flute, they sound ok, you do have to alter blowing pressure somewhat, nothing dramatically different in sound.  Interesting experiment.

Stan
